docs/scripts/model_feat_table.py

```diff
@@ -87,6 +87,14 @@
         "package": "langchain-huggingface",
         "link": "/docs/integrations/chat/huggingface/",
     },
+    "ChatNVIDIA": {
+        "tool_calling": True,
+        "json_mode": True,
```

Comment:
```suggestion
        "json_mode": False,
```
If the api supports json mode, could you link me where in the docs that's shown?